### PR TITLE
feat: Make Administration optional when updating a record

### LINKED ISSUE #818: Can't update project information without providing a new Canadian public administration
When selecting an existing project to update licence/tag information, because the Administration fields are required, the form can't be submitted.
I'm thinking that adding an administration might be something people want when updating a project, but maybe just have the heading with the "+" buton so other updates are possible

### PR DESCRIPTION
**Pull request by simon332114**
Updating a design/software/standard no longer requires an administration. Only applied to these 3 forms because code and partnership are organised differently as each project is associated to an admin file, so it wouldn't work if it wasn't required.

Fixes #818

### CODE REVIEW

**Comment by ShadeWyrm** [comment]
Field shows as required, we need to update the label to not be required when an Existing item is selected.

**Comment by simon332114** [comment]
My VM is stuck in the past but commit 0184e1b was made this morning in response to your comment

File: _data/design/government-of-canada-design-system.yml [hunk 1]
```diff
@@ -48,6 +48,6 @@ administrations:
         team:
           en: Digital Transformation Office
           fr: Bureau de la transformation numérique
-designStatus: 
-  en: Original design
-  fr: Design original
+        designStatus:
```
Comment (nschonni) [inline _data/design/government-of-canada-design-system.yml]: Is this supposed to be nested under the administration?

Comment (simon332114) [reply _data/design/government-of-canada-design-system.yml]: Yes, I made a mistake when originally setting it up, and it seemed easier to just tab it directly than resubmit the entire form

**Comment by ShadeWyrm** [comment]
@simon332114 - are we good to review this one?

**Comment by simon332114** [comment]
Yes

**Pull request merged**